Dockerfile.pai

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM ibmcom/powerai:1.6.1-base-ubuntu18.04-py3
+FROM ibmcom/powerai:1.6.2-base-ubuntu18.04-py37
```

Comment:
@mattborgard Do you have any idea why upgrading the base image causes [permission failure in creating a directory](https://travis-ci.com/IBM/MAX-Base/jobs/279485670#L308)? Anything changed here?

Reply:
My guess would be something changed, either intentionally or unintentionally, on the powerAI side. I'll ping that team to take a look.

Reply:
We changed to a non-root user(pwrai) in 1.6.2 as the default.
Normally this wouldn't cause a problem, except, by design I guess, docker creates new directories as `root:root`.  
A solution is to create the `/workspace` prior to the `WORKDIR` statement
`RUN sudo install -g pwrai -o pwrai -d /workspace`
Works on my end.  Let me know if you have any problems with this solution.   Thanks!!

Reply:
Thanks @dllehr81, your solution works! We also discovered that the path to anaconda has been changed to `/opt/anaconda/` in this new version (instead of `/opt/anaconda3/`).